bumblebee_status/modules/contrib/network_traffic.py

```diff
@@ -97,9 +97,6 @@ def default_network_adapter(cls):
         """Return default active network adapter"""
         gateway = netifaces.gateways()["default"]
 
-        if not gateway:
-            raise "No default gateway found"
-
```

Comment:
This conditional block is not useful because line #98 explodes if the `default` key is not found. :thinking: